src/sbmlutils/test/__init__.py

```diff
@@ -31,6 +32,993 @@
 
 REPRESSILATOR_SBML = MODELS_DIR / "repressilator" / "BIOMD0000000012_urn.xml"
 
+# Curated Biomodels
+BIOMD0000000001_SBML = MODELS_DIR / "biomodels_curated" / "BIOMD0000000001.xml"
```

Comment:
This is extreme code duplication. Only store the path to the directory here and create the test files automatically! See comments below